frontend/13f-app/src/components/CompanyFilingList.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react'
+import { useLocation, useParams, Link } from 'react-router-dom'
+import axios from 'axios'
+import './CompanyFilingList.css'
+
+function FilingsList () {
+  const [results, setResults] = useState([])
+  const { state } = useLocation()
+  const { companyId } = useParams()
+  useEffect(() => {
+    axios.get(`${process.env.REACT_APP_API_SERVER}/company/${companyId}/edgarfiling/`)
```

Comment:
```suggestion
    axios.get(`${process.env.REACT_APP_API_SERVER}/company/${companyId}/edgar-filing/`)
```
To match the aforementioned changes.